app/controllers/application_controller.rb

```diff
@@ -216,7 +216,7 @@ def find_locked_org_ids
       @service_request.sub_service_requests.each do |ssr|
         organization = ssr.organization
         if organization.has_editable_statuses?
-          self_or_parent_id = ssr.find_editable_id(organization.id)
+          self_or_parent_id = ssr.find_editable_id
```

Comment:
Removed parameter due to redundancy. See method refactor in SubServiceRequest.rb